tests/utils/test_databricks_utils.py

```diff
@@ -304,14 +305,23 @@ def get_context():
 
     with mock.patch(
         "dbruntime.databricks_repl_context.get_context",
-        return_value=mock.MagicMock(notebookId="notebook_id"),
+        return_value=mock.MagicMock(notebookId="notebook_id", notebookPath="/Repos/notebook_path"),
     ) as mock_get_context, mock.patch(
         "mlflow.utils.databricks_utils._get_property_from_spark_context"
     ) as mock_spark_context:
         assert databricks_utils.get_notebook_id() == "notebook_id"
-        mock_get_context.assert_called_once()
+        assert databricks_utils.is_in_databricks_repo_notebook()
+        assert mock_get_context.call_count == 2
```

Comment:
Hijacked this test a bit to minimize code duplication; can also copy paste it out